public_html/WEB-INF/jetty-env.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://www.eclipse.org/jetty/configure_10_0.dtd">
+
+<Configure id="wac" class="org.eclipse.jetty.webapp.WebAppContext">
+  <Set name="contextPath">/laptops</Set>
+  <Set name="war">/home/cloud_user/jetty/webapps/laptops.war</Set>
+  <New class="org.eclipse.jetty.plus.jndi.Resource">
+    <Arg><Ref refid="wac"/></Arg>
+    <Arg>jdbc/rssDS</Arg>
+     <Arg>
+        <Call class="oracle.ucp.jdbc.PoolDataSourceFactory" name="getPoolDataSource" >
+           <Set name="URL">jdbc:oracle:thin:@ils-db-prod.library.ucla.edu:1521:VGER</Set>
+           <Set name="user">ucla_preaddb</Set>
+           <Set name="password">ucla_preaddb_pwd</Set>
```

Comment:
Is this the actual password? Even though the account is read-only and access is limited to campus, I don't think we should check passwords in. We might want to change this now that it's been public. I also think this is something that would be better handled as a template in the Docker build and then let the private information be configured through ENV variables inside the container, rather than putting it into the war file.

Reply:
D'oh... would you like me to remove password from file or delete whole file from branch until we decide how to supply this info to Dockerized app?

Reply:
Removed jetty config and rebased; given password was for read-only, probably no need to change.